src/libostree/ostree-fetcher-curl.c

```diff
@@ -747,6 +747,16 @@ initiate_next_curl_request (FetcherRequest *req,
 
   if (self->tls_client_cert_path)
     {
+      if (!strncmp (self->tls_client_key_path, "pkcs11:", 7))
```

Comment:
Note there's a nice `g_str_has_prefix` function for this!

Reply:
Thanks!